src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml

```diff
@@ -0,0 +1,86 @@
+get:
+  summary: Get package commit history.
+  description: Get package commit history.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: deleted
+      schema:
+        type: string
+      description: Set to `1` to allow to retrieve the revision history of a deleted package.
+      example: 1
+    - in: query
+      name: limit
+      schema:
+        type: integer
+      description: Return the last `limit` elements of the revision history.
```

Comment:
```suggestion
      description: Limit the number of retrieved revision history elements to the specified number .
```

Reply:
@eduardoj just a suggestion, but for me it was not really clear from the description what it does.

Reply:
Sure. Applied.